libraries/Microsoft.Bot.Connector/OAuthClientConfig.cs

```diff
@@ -0,0 +1,120 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Net;
+using System.Net.Http;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Bot.Connector.Authentication;
+using Microsoft.Bot.Schema;
+using Microsoft.Rest;
+using Newtonsoft.Json;
+
+namespace Microsoft.Bot.Connector
+{
+    public static class OAuthClientConfig
+    {        
+        /// <summary>
+        /// The default endpoint that is used for API requests.
+        /// </summary>
+        public static string OAuthEndpoint { get; set; } = AuthenticationConstants.OAuthUrl;
```

Comment:
There seem like concurrency concerns here, as this would be read / written by any number of threads.